docs/guides/rate-limits-endpoint-based.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: endpoint-based-rate-limits
+title: Endpoint-Based Rate Limits
+sidebar_label: Endpoint-Based Rate Limits
+---
+
+## Introduction
+
+This page provides a comprehensive explanation of Ory's Endpoint-Based Rate Limiting. These limits are designed to fortify the
+security of your Ory project endpoints by mitigating common attack vectors such as brute-force and credential stuffing attempts,
+while allowing for flexibility for high-volume legitimate traffic from trusted sources.
+
+## What are Endpoint-Based Rate Limits?
+
+Endpoint-Based Rate Limits are precise controls applied to individual API endpoints within your Ory projects. Unlike Project-Based
+Rate Limits, which govern overall project request volumes, Endpoint-Based Rate Limits focus on safeguarding specific
+functionalities against abuse.
```

Comment:
I would rephrase this as system wide rules, which are menat to protect the system, and are a global limit which unless stated in an enterprise contract, cannot be broken regardless of project tier